mods/cnc/weapons/smallcaliber.yaml

```diff
@@ -114,85 +89,59 @@ M16:
 			Wood: 25
 			Light: 30
 			Heavy: 10
-		DamageTypes: Prone50Percent, TriggerProne, BulletDeath
 	Warhead@2Eff: CreateEffect
 		Explosions: piff
 
+Pistol:
+	Inherits: ^LightMG
+	ReloadDelay: 7
+	Range: 3c0
+	Report: gun18.aud
+	Warhead@1Dam: SpreadDamage
+		Damage: 1
+
+M16:
+	Inherits: ^LightMG
+
 MachineGun:
-	ReloadDelay: 20
+	Inherits: ^LightMG
 	Burst: 5
-	InvalidTargets: Wall
-	Range: 4c0
 	Report: mgun11.aud
-	Projectile: Bullet
-		Speed: 1c682
 	Warhead@1Dam: SpreadDamage
-		Spread: 128
-		Damage: 15
-		InvalidTargets: Wall
 		Versus:
-			None: 100
 			Wood: 10
 			Light: 50
-			Heavy: 10
-		DamageTypes: Prone50Percent, TriggerProne, BulletDeath
-	Warhead@2Eff: CreateEffect
-		Explosions: piffs
-
-Vulcan:
-	ValidTargets: Ground, Water
-	ReloadDelay: 2
-	Range: 6c0
-	Report: gun5.aud
-	Projectile: Bullet
-		Speed: 1c682
-	Warhead@1Dam: SpreadDamage
-		Spread: 426
-		Damage: 100
-		ValidTargets: Ground, Water
-		Versus:
-			None: 100
-			Wood: 25
-			Light: 100
-			Heavy: 35
-		DamageTypes: Prone50Percent, TriggerProne, BulletDeath
 	Warhead@2Eff: CreateEffect
 		Explosions: piffs
 
 APCGun:
 	ReloadDelay: 18
 	Range: 5c0
 	Report: gun20.aud
-	ValidTargets: Ground
 	Projectile: Bullet
 		Speed: 1c682
 	Warhead@1Dam: SpreadDamage
 		Spread: 128
 		Damage: 15
-		ValidTargets: Ground
 		Versus:
 			None: 50
 			Wood: 50
-			Light: 105
+			Light: 100
```

Comment:
Yes, because it doesn't change the effective damage (due to rounding), although I didn't consider veterancy tbh.